### ISSUE TITLE
How to transfrom .pt from .pth

### ISSUE THREAD

**Issue by guofenggitlearning** [LinhanDai/yolact-infer#1]
$ export PYTHONPATH="$PWD" && python models/export.py --weights ./weights/yolact.pt --img 640 --batch 1

I can't find models/export.py?
I try to modify the yolact.py,but failed!

**Comment by LinhanDai** [comment]
def evalimage(net:Yolact, path:str, save_path:str=None):
    frame = torch.from_numpy(cv2.imread(path)).cuda().float()
    batch = FastBaseTransform()(frame.unsqueeze(0))
    preds = net(batch)
    yalact_net = torch.jit.trace(net, batch)
    yalact_net.save('./yolact.pt')

把eval.py的函数修改成以上代码

**Comment by guofenggitlearning** [comment]
:Could not infer type of list element: Only tensors and (possibly nested) tuples of tensors, lists, or dictsare supported as inputs or outputs of traced functions, but instead got value of type Yolact.
![yolact](https://user-images.githubusercontent.com/42864810/95642204-fc481a80-0a74-11eb-887b-0090d2100ba1.jpg)

**Comment by LinhanDai** [comment]
yolact.py文件 ，返回替换一下
class Yolact(nn.Module):
#return self.detect(pred_outs, self)
return (pred_outs["loc"], pred_outs['conf'], pred_outs['mask'], pred_outs['proto'])
